test/test_tensorclass.py

```diff
@@ -527,6 +622,14 @@ class MyDataParent:
     assert data.tensordict["X"] is X_clone
     data.z = td_clone
     assert data.tensordict["z"] is td_clone
+    v_new = "test_bluff"
+    data.v = v_new
+    assert data.v == v_new
+    assert data.y.v == "test_nested"
+    data.y.v = "test_nested_new"
+    assert data.y.v == "test_nested_new"
```

Comment:
is `data_nest.v` changed too here?

Reply:
No. data_nest.v would remain the same(" test_nested"). Do we want to change the behavior here?